src/mongo/db/query/explain.cpp

```diff
@@ -289,6 +289,15 @@ unique_ptr<PlanStageStats> getWinningPlanStatsTree(const PlanExecutor* exec) {
                : std::move(exec->getRootStage()->getStats());
 }
 
+
+/**
+ * Get PlanExecutor's original winning plan stats tree.
```

Comment:
Please elaborate here that this will only be different from the actual winning plan if a non-blocking backup plan was used. How about `Returns the root of the original winning plan used by 'exec'. This may be different than the final winning plan in the case that the MultiPlanStage selected a blocking plan which failed, and fell back to a non-blocking plan instead. If there is no MultiPlanStage in the tree, returns the root stage of 'exec'.`